server/graphql/v1/mutations.js

```diff
@@ -204,12 +206,25 @@ const mutations = {
           throw new Error('User already exists for given email');
         }
 
+        if (
```

Comment:
I don't think it's useful to do any modification in these GraphQL queries. The `signin` endpoint should be enough.

Reply:
This is for creating a new profile with blockstack.

Reply:
Ok, sorry, I was a bit wrong. `createUser` was indeed used for signups.

But, please handle that in a separate endpoint (ie: blockstackSignin / blockstackSignup), or a separated GraphQL mutation (ie: createUserWithBlockstack).

Reply:
I have re-added it and moved everything into separate methods.